numpy/lib/tests/test_arraypad.py

```diff
@@ -82,6 +125,30 @@ def test_check_maximum_2(self):
             )
         assert_array_equal(a, b)
 
+    def test_check_maximum_stat_length(self):
```

Comment:
The formatting of the test results is clear, but it is going to make for a lot of PEP8 violations.
```
36      E201 whitespace after '['
30      E203 whitespace before ','
499     E241 multiple spaces after ','
```

I wonder if there is another way to express the results that is as clear but doesn't have this problem? For instance, this could be
```
b = np.array([10]*25 + range(1,101) + [100]*20)
```